SwiftTweaks/TweakCallbacks.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  TweakCallbacks.swift
+//  SwiftTweaks
+//
+//  Created by Jarosław Pendowski on 28/04/2017.
+//  Copyright © 2017 Khan Academy. All rights reserved.
+//
+
+import Foundation
+
+public typealias TweakBlock = () -> Void
+
+public class TweakCallbacks {
```

Comment:
So my logic was that although the action/closure tweak lives with other tweaks it's very different from them, so it should be handled by a different construct.

But maybe there's an argument there that it could be easier for users to use just one method, so I started moving the logic and confirmed what I thought those couple months ago. It would require a major rewrite of the architecture. It's the only tweak (for now), that doesn't really have a value and a decent amount of logic in Tweaks assumes that there is one - if not the current one then at least a default one you have to provide. So that would probably mean that we should make `defaultValue` and all methods depending on it returning an `Optional`, which complicates the code on many levels. And that's just one issue. 
Another difference would be that `bind` wouldn't be called for `closure` tweak type. 
And lastly - we still need `TweakCallbacks` (or any type) for proper generics support for creating Tweaks.

It is, of course, doable but I'm not sure if it's worth to avoid a separate type (to some degree - again - generics from above), which does provide an extra functionality anyway (adding/removing callbacks when needed).